### ISSUE TITLE
Addition of a config file to store default options?

### ISSUE THREAD

**Issue by saurabh-deochake** [neo1691/scorer.py#38]
As @neo1691 suggested that we should add command line options to the script. It is a good idea but I think we should mention all default values in a config file located at the app dir or /etc/ something. 

We can write a sample file like below.

/***** /etc/scorer/config *****/
[config]
config_file = "/path/to/config/file"

[log]
log_file = "/path/to/log/file"
log_level = "debug" 

[app]
delay = 5

[secret]
api_key = 'xxxxxxxxx'
access_token = "xxxxxxxxxxx"

This way it not only makes code cleaner but also makes it easy to change the default values (using ConfigParser) through command line options or code itself.

What do you guys think about the config file option?

**Comment by neo1691** [comment]
I agree with this, by default the script will read options defined from a config file UNLESS the user overrides them by providing the command line arguments. 

We can keep a global config in /etc/ and we can also keep a user level config in $XDG_HOME which is usually ~/.config/

**Comment by nirmalvp** [comment]
Agreed
